### PR TITLE
Various Additions

### LINKED ISSUE #47: additional block explorers
- https://stake.id/#/
- https://www.cosmostation.io/

### LINKED ISSUE #49: add swift sdk
https://github.com/CosmosSwift

### LINKED ISSUE #59: add provenance.io
chain running cosmos-sdk

http://provenance.io/

### LINKED ISSUE #62: add sharering
https://github.com/ShareRing/Shareledger

### LINKED ISSUE #66: add stakebird
https://github.com/public-awesome/stakebird

### PR DESCRIPTION
**Pull request by marbar3778**
- closes #47 
- closes #59 
- closes #62 
- closes #66 
- closes #49

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -134,6 +136,13 @@ If you are working on a module that you would like to be used by the community,
 - [Node Monitor Gaiad by Stake Zone](https://github.com/stakezone/nodemonitorgaiad) - Log file based Cosmos `gaiad` monitoring solution for Zabbix.
 - [Cosmosvalmon by Fresh Atoms](https://github.com/gaia/cosmosvalmon) - Configurable Cosmos validator monitoring and alerting service.
 
+## Framework
```
Comment (marbar3778) [inline README.md:139]:
```suggestion
## Frameworks
```

**Comment by marbar3778** [comment]
@hxrts I cant tag you to help review this. Do you have perms on this repo?

File: README.md [hunk 2]
```diff
@@ -134,6 +136,13 @@ If you are working on a module that you would like to be used by the community,
 - [Node Monitor Gaiad by Stake Zone](https://github.com/stakezone/nodemonitorgaiad) - Log file based Cosmos `gaiad` monitoring solution for Zabbix.
 - [Cosmosvalmon by Fresh Atoms](https://github.com/gaia/cosmosvalmon) - Configurable Cosmos validator monitoring and alerting service.
 
+## Frameworks
+
+- [Cosmos-SDK](https://github.com/cosmos/cosmos-sdk/)
```
Comment (okwme) [inline README.md:141]:
```suggestion
- [Cosmos-SDK](https://github.com/cosmos/cosmos-sdk/) (GoLang)
```

File: README.md [hunk 3]
```diff
@@ -134,6 +136,13 @@ If you are working on a module that you would like to be used by the community,
...
+- [Kepler](https://github.com/f-o-a-m/kepler)
```
Comment (okwme) [inline README.md:142]:
```suggestion
- [Kepler](https://github.com/f-o-a-m/kepler) (Haskell)
```

File: README.md [hunk 4]
```diff
@@ -134,6 +136,13 @@ If you are working on a module that you would like to be used by the community,
...
+- [Orga](https://github.com/nomic-io/orga)
```
Comment (okwme) [inline README.md:143]:
```suggestion
- [Orga](https://github.com/nomic-io/orga) (Rust)
```

File: README.md [hunk 5]
```diff
@@ -134,6 +136,13 @@ If you are working on a module that you would like to be used by the community,
...
+- [CosmosSwift](https://github.com/CosmosSwift)
```
Comment (okwme) [inline README.md:144]:
```suggestion
- [CosmosSwift](https://github.com/CosmosSwift) (Swift)
```

**Comment by hxrts** [comment]
> @hxrts I cant tag you to help review this. Do you have perms on this repo?

nope

File: README.md [hunk 6]
```diff
@@ -134,6 +136,13 @@ If you are working on a module that you would like to be used by the community,
...
+
```
Comment (okwme) [inline README.md:145]:
```suggestion
- [Loton.js](https://github.com/nomic-io/lotion) (Javascript)

```

Comment (marbar3778) [reply README.md:145]: Is it still maintained?

Comment (okwme) [reply README.md:146]: It still works although not currently maintained. We have been speaking w a team about taking over though.

Comment (marbar3778) [reply README.md:146]: AMAZING!!

**Pull request merged**